tests/jquery.flot.navigate.Test.js

```diff
@@ -278,6 +279,66 @@ describe("flot navigate plugin", function () {
         });
     });
 
+    describe('touchDrag', function() {
+      it('should drag the plot (start + move + end)',function() {
+
+        plot = $.plot(placeholder, [
+            [
+                [0, 0],
+                [10, 10]
+            ]
+        ], options);
+
+        xaxis = plot.getXAxes()[0];
+        yaxis = plot.getYAxes()[0];
+
+        sendTouchEvent(plot.getXAxes()[0].p2c(5),plot.getYAxes()[0].p2c(5),placeholder[0].childNodes[2],"touchstart");
+        sendTouchEvent(plot.getXAxes()[0].p2c(6),plot.getYAxes()[0].p2c(6),placeholder[0].childNodes[2],"touchmove");
+        sendTouchEvent(0,0,placeholder[0].childNodes[2],"touchend");
+
+        expect(xaxis.min).toBeCloseTo(-1,2);
+        expect(yaxis.max).toBeCloseTo(9,2);
+        expect(xaxis.max).toBeCloseTo(9,2);
+        expect(yaxis.min).toBeCloseTo(-1,2);
+
+      });
+    });
+
+  function sendTouchEvent(x, y, element, eventType) {
```

Comment:
move this function to tests/utils/simulate.js
and create 
simulate.touchStart
simulate.touchMove
simulate.touchEnd

and 

simulate.touchDrag that call all three